sunpy/map/tests/test_mapsequence.py

```diff
@@ -169,21 +168,18 @@ def test_repr_html(mapsequence_all_the_same):
         assert m._repr_html_() in html_string
 
 
-def test_quicklook(mapsequence_all_the_same):
-    with mock.patch('webbrowser.open_new_tab') as mockwbopen:
-        mapsequence_all_the_same.quicklook()
-
+def test_quicklook(mocker, mapsequence_all_the_same):
+    mockwbopen = mocker.patch('webbrowser.open_new_tab')
```

Comment:
Using mocker we can do this, if we want to replace all of the mocks, we would need to follow this pattern inside the  test as far as I can tell.

Reply:
Its ok.